extensions/content-manager/admin/src/components/CustomTable/Row.js

```diff
@@ -71,7 +72,7 @@ const getDisplayedValue = (type, value, name) => {
   }
 };
 
-function Row({ goTo, isBulkable, row, headers, level, relationColumnName }) {
+function Row({ goTo, isBulkable, row, headers, level, relationColumnName, isWrappedWithRoot, manageCollapsedIds, isCollapsed }) {
```

Comment:
I will wrap it to next line fro better readability

Reply:
fixed